common/presenters/allocations-to-table-component.js

```diff
@@ -27,11 +27,15 @@ function _byAdded(totalSlots, unfilledSlots) {
 function allocationsToTableComponent({
   showFromLocation = false,
   showRemaining = false,
+  query,
+  isSortable = true,
```

Comment:
Just realised it looks like there is no test to test the default scenario and the scenario where this is overridden to `false`. Could we add some tests for this flag please.